### ISSUE TITLE
History of wearing tightly fitted clothing

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9967]
**Preferred term label:**
History of wearing tightly fitted clothing

**Synonyms**
Tight clothing, compression shorts, sports bras, bike shorts, wet suits, spandex, veils, tight hats

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tinea corporis, tinea cruris,  Black piedra

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000794
- label: History of wearing tightly fitted clothing
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: History of wearing tight clothing such as compression shorts, sports bras, bike shorts, wet suits, spandex, veils, and tight hats.
- comment: This may be a risk factor for tinea.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9967
- synonyms:
